src/compressed_tensors/compressors/model_compressors/model_compressor.py

```diff
@@ -276,8 +277,9 @@ def compress(
                 )
 
         if self.sparsity_compressor is not None:
+            compression_targets = self._find_sparse_compression_targets(model=model)
```

Comment:
do we not care about adding something similar like a sparsity scheme, similar to a quantization scheme, for each layer we want to target, describing how the layer was targeted?

Reply:
Agree we should be adding a sparsity config -- ideally we would have one compression config that represents the state of a given module/param and saves how it was encoded and therefore how to load it